### DISCUSSION TITLE
[4.0]Layout override not working for com_contact/category

### DISCUSSION THREAD

**Discussion by shawnhy** [joomla/joomla-cms#31697] [Q&A] [answered]
### Steps to reproduce the issue

In `/layouts/joomla/content/category_default.php`,I edit line62 to`<h2 class="origin">`
Then, copy file: `/layouts/joomla/content/category_default.php` to `/templates/cassiopeia/html/layouts/content/category_default.php`
and edit line62 `<h2>` to `<h2 class="override">`

### Expected result
I expect to find class "override" added to h2 title category list page of contact. 

### Actual result

It is still using `/layouts/joomla/content/category_default.php`.

### System information (as much as possible)
php7.2.34

Joomla4.0 beta 4

### Additional comments
I did the same in Joomla3.9 and work.

**Comment by chmst** [comment, marked as answer]
The override file has to be in /templates/cassiopeia/html/layouts/ **joomla** / content/category_default.php. For overrides, please use the feature "create overrides" in template styles.
